cmds/rush/tty_linux.go

```diff
@@ -0,0 +1,59 @@
+
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"unsafe"
+)
+
+const (
+	SYS_ioctl         = 16
+)
+func tty() {
```

Comment:
The intent of this function is to do "whatever" we will need to get the tty in the correct state. For now, it's one thing, but I expect we'll find more.